app/frontend/src/styles/application/jobseekers/vacancy.scss

```diff
@@ -14,12 +14,27 @@
     }
   }
 
-  .similar-jobs {
-    .govuk-grid-row {
-      align-items: stretch;
-      display: flex;
-      flex-wrap: wrap;
-      margin: 0;
+  .similar-job {
+    border-left: 5px solid govuk-colour('orange');
+    margin-bottom: govuk-spacing(4);
+    padding-bottom: govuk-spacing(2);
+    padding-left: govuk-spacing(2);
+    padding-right: govuk-spacing(4);
+    padding-top: govuk-spacing(2);
+  }
+
+  .similar_jobs_position_test {
```

Comment:
id also favour a mobile first kind of thing for this. whilst the benefit is neither here or there for ab test, its a better habit to be in